view/frontend/web/js/view/payment/abstract-method.js

```diff
@@ -152,6 +132,56 @@ define([
             window.addEventListener('onError', function (event) {
                 console.log(event.detail);
             });
+        },
+
+        createIntent: function () {
+            const method = this.isChecked();
+            const payload = {
+                'method': method
+            }
+            $.ajax({
+                url: url.build('rest/V1/airwallex/payments/create_intent'),
+                method: 'POST',
+                contentType: 'application/json',
+                async:false,
+                data: JSON.stringify(payload),
+                success: function (result) {
+                    this.responseData = JSON.parse(result);
+                    this.intentId(this.responseData.id);
+                }.bind(this),
+                error: function () {
+                    globalMessageList.addErrorMessage({
+                        message: $.mage.__('An error occurred on the server. Please try to place the order again.'),
+                    });
+
+                    $('body').trigger('processStop');
+                }
+            });
+        },
+
+        refreshIntent: function () {
+            const payload = {
+                'intentId': this.intentConfiguration().id,
```

Comment:
Same as previous comment. Can also use object destructuring.

Reply:
will proceed to fix this